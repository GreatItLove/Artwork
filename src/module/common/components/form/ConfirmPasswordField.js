import React from 'react';

import { Field } from 'redux-form';

import InputField from './InputField';

/**
 * @function ConfirmPasswordField
 * @param {Object} props input props
 * @returns {XML}
 */
function ConfirmPasswordField(props) {
  return (
    <Field
      id="confirmPassword"
      name="confirmPassword"
      type="password"
      fullWidth={true}
      component={InputField}
      autoComplete="new-confirm-password"
      {...props}
    />
  );
}

export default ConfirmPasswordField;
