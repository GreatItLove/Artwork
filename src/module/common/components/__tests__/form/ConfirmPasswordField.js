import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import ConfirmPasswordField from '../../form/ConfirmPasswordField';

describe('Confirm Password Field', () => {
  it('Snapshot of ConfirmPasswordField', () => {
    const wrapper = shallow(<ConfirmPasswordField />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
