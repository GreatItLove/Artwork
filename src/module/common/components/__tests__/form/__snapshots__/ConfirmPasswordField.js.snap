// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirm Password Field Snapshot of ConfirmPasswordField 1`] = `
<Field
  autoComplete="new-confirm-password"
  component={[Function]}
  fullWidth={true}
  id="confirmPassword"
  name="confirmPassword"
  type="password"
/>
`;
